.github/actions/verify-dotnet-consolidation/action.yml

```diff
@@ -20,7 +20,8 @@ runs:
   steps:
     - name: Setup
       shell: pwsh
-      run: dotnet tool install dotnet-consolidate --global --version 2.0.0
+      run: |
+        & "${{ github.action_path }}/../../../Scripts/Install-DotNetTool.ps1" -Name "dotnet-consolidate" -Version "2.0.0" -Global
```

Comment:
Now that we have a Scripts folder, please also move Check-PullRequestTitle.ps1 to it.